101-servicebus-create-namespace-geo-recoveryconfiguration/README.md

```diff
@@ -0,0 +1,9 @@
+# 101-servicebus-create-namespace
```

Comment:
add something to the readme... about the sample.

Reply:
Added a short description.